xcell/cls/theory.py

```diff
@@ -119,10 +119,49 @@ def compute_tracer_ccl(self, name, tracer, mapper):
                 'with_hm': with_hm,
                 'normed': normed_profile}
 
+    def get_ccl_tkka(self, ccl_trA1, ccl_trA2, ccl_trB1, ccl_trB2,
+                     kind='1h'):
+        if kind not in ['1h']:
+            raise NotImplementedError(f"Non-Gaussian term {kind} "
+                                      "not supported.")
+
+        pA1 = ccl_trA1['ccl_pr']
+        pA2 = ccl_trA2['ccl_pr']
+        if ccl_trA1['name'] == ccl_trA2['name']:
+            pr2ptA = ccl_trA1['ccl_pr_2pt']
+            pA2 = pA1
+        else:
+            pr2ptA = None
+        pB1 = ccl_trB1['ccl_pr']
+        pB2 = ccl_trB2['ccl_pr']
+        if ccl_trB1['name'] == ccl_trB2['name']:
+            pr2ptB = ccl_trB1['ccl_pr_2pt']
+            pB2 = pB1
+        else:
+            pr2ptB = None
+
+        k_s = np.geomspace(1E-4, 1E2, 512)
```

Comment:
Is it ok having it hardcoded?

Reply:
For all supported tracers this is accurate enough while being manageable (everything computes in under 1s). Note we're already doing the same thing for the halo model power spectra.

We can make it fully tunable in the future if we need to.